test/extern/kushal/extern-includes.future

```diff
@@ -0,0 +1,28 @@
+bug: Unable to include external non-standard header files
```

Comment:
In this description, I would call out that the problem only occurs with the incremental flag